tgui/packages/tgui/interfaces/TankDispenser.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright (c) 2020 @actioninja
+ * Minor changes by Aleksej Komarov
+ * SPDX-License-Identifier: MIT
+ */
+
+import { useBackend } from '../backend';
+import { Button, LabeledList, Section } from '../components';
+import { Window } from '../layouts';
+
+export const TankDispenser = (props, context) => {
+  const { act, data } = useBackend(context);
+  return (
+    <Window
+      width={275}
+      height={105}>
+      <Window.Content>
+        <Section>
+          <LabeledList>
+            <LabeledList.Item
+              label="Plasma"
+              buttons={(
+                <Button
+                  icon={data.plasma ? 'circle' : 'circle-o'}
+                  content="Dispense"
+                  disabled={!data.plasma}
+                  onClick={() => act('plasma')} />
```

Comment:
As this is intended as a "tutorial" component, thoughts on naming the action `dispense-plasma` instead of just `plasma`? I know what it means, you know what it means, but someone coming in might thing it's related to the `plasma` value on the `data` object or something.

Reply:
ye ye